### DISCUSSION TITLE
how to create databases for the licata city app, the tourist promotion app of the city

### DISCUSSION THREAD

**Discussion by AngeloSanf** [supabase/supabase#5780] [Questions] [answered]
How can I create the database containing categories such as beaches, monuments etc and for each category insert title, photo, description and coordinates? Thanks in advance, I've never done a database, I've just started programming.

**Comment by GaryAustin1** [comment, marked as answer]
You really should just google around for postgres, sql or relationship database tutorials and see how people set them up and use them first.
You should be able to find some on info collection type apps.
Supabase has some starter info, but most assume you know how to at least code an app and are a bit familiar with databases.
https://supabase.com/docs/guides/examples

**Reply by AngeloSanf** [reply]
Thanks
